File: java/src/main/java/org/example/stack/StackRun.java
package org.example.stack;

public class StackRun {
    public static void run() {
        var stack = new Stack();
        System.out.println(stack.empty());
        stack.push("casas");
        stack.push("novas");
        stack.push("listas");

        System.out.println(stack.empty());
        System.out.println(stack.top());

        stack.pop();
        stack.pop();
        stack.pop();
        System.out.println(stack.empty());
    }
}

File: php/src/Stack/StackRun.php
<?php

namespace Claudiosilvajunior\Php\Stack;

class StackRun
{
    public static function run(): void
    {
        $stack = new Stack();
        var_dump($stack->empty());
        $stack->push("house1");
        $stack->push("house2");
        $stack->push("house3");
        $stack->push("house4");
        var_dump($stack->top());

        $stack->pop();
        var_dump($stack->empty());
        var_dump($stack->top());
    }
}